README.md

```diff
@@ -118,64 +118,21 @@ To add resources or to contribute read the [Contribution-guide](https://github.c
 ---  
 
 ## Angular 
-1. IDEs
+- IDEs
 
     - [visualstudio](https://visualstudio.microsoft.com/downloads/)
     - [visualstudioCode](https://code.visualstudio.com/Download)
-
-2. HTML
-
-    - [w3schools](https://www.w3schools.com/html/) HTML Tutorial
-    - [MDN Web Docs](https://developer.mozilla.org/en-US/docs/Web/HTML) HTML: HyperText Markup Language
-
-3. CSS
-
-    - [w3schools](https://www.w3schools.com/css/) CSS Tutorial
-    - [MDN Web Docs](https://developer.mozilla.org/en-US/docs/Web/CSS) CSS: Cascading Style Sheets
-    - [fontawesome](https://fontawesome.com/) Official website
-    - [w3schools](https://www.w3schools.com/icons/icons_reference.asp) Icons Reference
-
-4. JavaScript
-
-    _Basic_
-
-    - [w3schools](https://www.w3schools.com/js/default.asp) JavaScript Tutorial
-    - [developer.mozilla](https://developer.mozilla.org/en-US/docs/Web/JavaScript) JavaScript
-    - [javascript.info](https://javascript.info/) The Modern JavaScript Tutorial
-    - [htmlcheatsheet](https://htmlcheatsheet.com/js/) JS CheatSheet
-
-    _es6_
-
-    - [exploringjs](https://exploringjs.com/es6/index.html) Exploring ES6
-    - [es6-features](http://es6-features.org/#Constants) ECMAScript 6 — New Features: Overview & Comparison
-    - [jsfeatures](https://jsfeatures.in) jsfeatures
-    - [babeljs](https://babeljs.io/docs/en/learn) Learn ES2015
-    - [github](https://github.com/lukehoban/es6features) ECMAScript 6
-    - [github](https://github.com/sudheerj/ECMAScript-features) ECMAScript 6
-
-    _Style Guide_
-
-    - [github](https://github.com/airbnb/javascript) Airbnb JavaScript Style Guide
-    - [github](https://github.com/ryanmcdermott/clean-code-javascript) clean-code-javascript
-
-    _playground_
-
-    - [js-array-playground](https://js-array-playground.firebaseapp.com/) JavaScript Array Playground
-    - [jsfiddle](https://jsfiddle.net/)
-    - [plnkr](https://plnkr.co/)
-    - [jsbin](https://jsbin.com/?html,output)
-    - [codepen](https://codepen.io/pen/)
     - [stackblitz](https://stackblitz.com/)
 
-5. Angular
+- Angular
 
     - [official website](https://angular.io/)
```

Comment:
please remove these one line spaces in between the heading and its related resources, from the rest of your added resources